### ISSUE TITLE
getRawTransaction RPC return error for some transactions 

### ISSUE THREAD

**Issue by dindinw** [Qitmeer/qitmeer#257]
For the following transactions 
```
11e556600b7f44a9e7a6f4ecaca1646b3f7d635d99d6a802ea78a5c594714de3
12cefcf5ef76e247f0ca68c4f3f71d8c8b3c0ab1f659682a8cde38431446d808
34dbc68909d8648c071d048c9dad999cf2021358f896fc0869f712ed94116a0e
4c13ab17a86bfa1c4b8529961efc46179255425c5038169a81b5dd45a49cad80
```
The `getRawTransaction` RPC return error when try to retrieve detail by using `txId`

```
curl -s -k -u test:test -X POST -H 'Content-Type: application/json' --data '{"jsonrpc":"2.0","method":"getRawTransaction","params":["11e556600b7f44a9e7a6f4ecaca1646b3f7d635d99d6a802ea78a5c594714de3",true],"id":1}' https://127.0.0.1:18131
{"jsonrpc":"2.0","id":1,"error":{"code":-32000,"message":"No information available about transaction 11e556600b7f44a9e7a6f4ecaca1646b3f7d635d99d6a802ea78a5c594714de3"}}
```

**Comment by lochjin** [comment]
> For the following transactions
> 
> ```
> 11e556600b7f44a9e7a6f4ecaca1646b3f7d635d99d6a802ea78a5c594714de3
> 12cefcf5ef76e247f0ca68c4f3f71d8c8b3c0ab1f659682a8cde38431446d808
> 34dbc68909d8648c071d048c9dad999cf2021358f896fc0869f712ed94116a0e
> 4c13ab17a86bfa1c4b8529961efc46179255425c5038169a81b5dd45a49cad80
> ```
> 
> The `getRawTransaction` RPC return error when try to retrieve detail by using `txId`
> 
> ```
> curl -s -k -u test:test -X POST -H 'Content-Type: application/json' --data '{"jsonrpc":"2.0","method":"getRawTransaction","params":["11e556600b7f44a9e7a6f4ecaca1646b3f7d635d99d6a802ea78a5c594714de3",true],"id":1}' https://127.0.0.1:18131
> {"jsonrpc":"2.0","id":1,"error":{"code":-32000,"message":"No information available about transaction 11e556600b7f44a9e7a6f4ecaca1646b3f7d635d99d6a802ea78a5c594714de3"}}
> ```

https://github.com/Qitmeer/qitmeer/pull/258

**Comment by dindinw** [comment]
fixed by #258
